docs/examples/GradleExamples/GradleJava7Example/build.gradle

```diff
@@ -53,10 +48,5 @@ allprojects {
                 //'-Awarns',
                 "-Xbootclasspath/p:${configurations.checkerFrameworkAnnotatedJDK.asPath}"
         ]
-        if (targetJavaVersion.isJava7()) {
-            compile.options.compilerArgs += ['-source', '7', '-target', '7']
-            options.fork = true
-            options.forkOptions.jvmArgs += ["-Xbootclasspath/p:${configurations.checkerFrameworkJavac.asPath}"]
```

Comment:
This example uses annotations in comments, so it needs to use jsr308 javac.

Reply:
Also, this example is duplicated in the manual, so the manual example also needs to be updated.

Reply:
I think there are two issues:
1. The way that the `targetJavaVersion` is determined with:  `ext.targetJavaVersion = JavaVersion.current().isJava7()` makes it depend on the JDK that is used to run the compiler. We now require Java 8 to run the Checker Framework, so this can never be 7, right?
2. Even if we want the jsr308 javac, we don't need it on the bootclasspath and it already is on the normal classpath.

So don't we need a new way to decide whether to pass `source/target 7` ?

Reply:
Right.  So I guess this example should just always pass -source 7 -target 7?

Reply:
I'm a bit confused why the original code had the conditional check, if this should always be a Java 7 example.
I'm fine with just always adding `-source 7 -target 7`.

Reply:
I guess we didn't realize it was using annotations in comments.  I'll make the change.

Reply:
Annotations in Comments don't need source/target 7.

Reply:
Let's continue this discussion on #1484.